### PR TITLE
bump version to 0.2.0

### PR DESCRIPTION
**Pull request by jpchen**
Blocked by #1092

0.2!

### CODE REVIEW

File: pyro/primitives.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 from pyro.poutine.runtime import _DIM_ALLOCATOR, _MODULE_NAMESPACE_DIVIDER, _PYRO_PARAM_STORE, am_i_wrapped, apply_stack
 from pyro.util import deep_getattr, set_rng_seed  # noqa: F401
 
-version_prefix = '0.2.0-a0'
+version_prefix = '0.2.0'
```
Comment (neerajprad) [inline pyro/primitives.py]: Not part of this PR, but we should remove this from `primitives.py`. I don't think its needed.

Comment (fritzo) [reply pyro/primitives.py]: I believe this should be in `__init__.py`

Comment (fritzo) [reply pyro/primitives.py]: Actually, can we remove it from primitives.py since it's already in `__init__.py`?
